docs/development.md

```diff
@@ -59,22 +59,30 @@ bundle exec foreman start
 
 ### Stable Katello Development Box
 
-A stable box image of a fully installed Katello development environment is published to Vagrant cloud nightly (on successful installs). In the `vagrant/boxes.d/99-local.example` file, a box using this image is available under the name `centos7-katello-devel-stable`. If it's the first time spinning this box up, you can do:
+#### Using the stable Katello development box
 
-```
-vagrant up centos7-katello-devel-stable
-```
+When spinning up a Katello development environment locally, it can take a while to install and isn't always guaranteed finish successfully. A stable box image of a fully installed Katello development environment is created nightly and [is published to Vagrant cloud](https://app.vagrantup.com/katello/boxes/katello-devel) when the install succeeds. This image will have a Katello development environment successfully installed, ready to use.
+
+In the `vagrant/boxes.d/99-local.example` file, a box using this image is available under the name `centos7-katello-devel-stable`. If it's the first time spinning this box up, you can do the following to create the box:
 
-This will spin up a Katello development environment in the time it takes to download from Vagrant cloud. At this moment, you will have to manually configure any personal customizations like github forks.
+1. Copy `vagrant/boxes.d/99-local.yaml.example` to `vagrant/boxes.d/99-local.yaml`. If you already have a `99-local.yaml`, you can copy the entries in `99-local.yaml.example` to your `99-local.yaml`.
+2. `vagrant up centos7-katello-devel-stable` to spin up the box
 
-After you have spun the box up once, if you would like a new box with the latest box image availble, you will need to destroy the existing box and update the underlying box image before spinning up the new box.
+This will spin up a Katello development environment in the time it takes to download from Vagrant cloud. At this moment, you will have to manually configure any personal customizations such as github remotes.
 
-For example:
-```
-vagrant destroy centos7-katello-devel-stable
-vagrant box update centos7-katello-devel-stable
-vagrant up centos7-katello-devel-stable
-```
+#### Creating a new Katello development box with the latest box image
+
+*Note: the following only is applicable after you have already spun up a `centos7-katello-devel-stable` box. If this is your first time spinning one up, only follow the instructions above*
+
+Once you have spun up the `centos7-katello-devel-stable` box once, you will want to update the box image used by Vagrant before spinning up another `centos7-katello-devel-stable` box. This is because Vagrant will use the latest image downloaded locally and you will want to use the latest image published to Vagrant cloud. This workflow is useful when you want an updated "fresh" environment. You can run the following to destroy your existing box, upgrade the box image to the latest one available, and spin up a new box.
```

Comment:
You can drop the 'This is because' it creates a bit of a run on. I think we should emphasize not simply a fresh environment but that the environment is changing as the Foreman evolves. Newer versions of backend systems, newer services and deployment changes are reflected in the new images and developers should want to be running the latest to ensure their new features work.